### ISSUE TITLE
Remove HP_0000118 (phenotypic abnormality) subclass of UPHENO_0001002?

### ISSUE THREAD

**Issue by matentzn** [obophenotype/human-phenotype-ontology#4325]
I dont know what would break if we did that, but I think this axiom is better served by upheno then hp. it also contributes to a slightly odd looking class hierarchy in HP. Any concerns moving this axiom into upheno?

**Comment by drseb** [comment]
it is just a subclass axiom between an HP class and a upheno class. Does not belong in HP AFAIK

**Comment by matentzn** [comment]
Exactly. Right now phenotypic abnormality is a subclass of the HP:All class AND that Upheno class. It would be sufficient if phenotypic abnormality was simply under Hp:All, we think!

**Comment by drseb** [comment]
@pnrobinson also ok from your side to remove this?

**Comment by pnrobinson** [comment]
1+
